packages/shared/src/components/modals/ClickbaitShieldModal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { type MouseEvent, type ReactElement } from 'react';
+import { Modal, type ModalProps } from './common/Modal';
+import { Image } from '../image/Image';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../typography/Typography';
+import { PlusUser } from '../PlusUser';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { webappUrl } from '../../lib/constants';
+import { DevPlusIcon } from '../icons';
+import { usePlusSubscription, useViewSize, ViewSize } from '../../hooks';
+import { LogEvent, TargetId } from '../../lib/log';
+import { Switch } from '../fields/Switch';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { clickbaitShieldModalImage } from '../../lib/image';
+
+type Props = {
+  hasUsedFreeTrial?: boolean;
+  fetchSmartTitle?: () => Promise<void>;
+};
+
+const ClickbaitShieldModal = ({
+  hasUsedFreeTrial = true,
+  fetchSmartTitle,
+  ...props
+}: Props & ModalProps): ReactElement => {
+  const { logSubscriptionEvent } = usePlusSubscription();
+  const { closeModal } = useLazyModal();
+  const isMobile = useViewSize(ViewSize.MobileL);
+  if (!isMobile) {
+    closeModal();
+    return null;
+  }
```

Comment:
Why is this needed?

Reply:
Because we don't want to show the modal on other devices